frontend/src/app/modules/external-booking/external-booking-choose-time-page/external-booking-choose-time-page.component.ts

```diff
@@ -243,12 +260,23 @@ export class ExternalBookingTimeComponent extends BaseComponent implements OnIni
         );
     }
 
-    addSlotInfo(slotId: bigint, teamId: bigint | undefined, duration: number, location: LocationType, name: string, meetingRoom?: string) {
+    addSlotInfo(
+        slotId: bigint,
+        teamId: bigint | undefined,
+        duration: number,
+        location: LocationType,
+        name: string,
+        meetingRoom?: string,
+    ) {
         this.selectedDurationAndLocationEvent.emit({ slotId, teamId, duration, location, name, meetingRoom });
     }
 
     redirectToChooseMeeting() {
         this.reloadData.emit(this.userLink);
         this.router.navigateByUrl(`/external-booking/choose-meeting/${this.userLink}`);
     }
+
+    override ngOnDestroy(): void {
+        super.ngOnDestroy();
+    }
 }
```

Comment:
This does nothing as far as I'm concerned

Reply:
Fixed